### ISSUE TITLE
Nixio 1.5.0b4 breaks unittests

### ISSUE THREAD

**Issue by JuliaSprenger** [NeuralEnsemble/python-neo#778]
With the switch from nixio version 1.5.0b3 to 1.5.0b4 the unittests are breaking for the NixRawIO.
The error is
```
Traceback (most recent call last):
  File "/home/circleci/repo/neo/test/rawiotest/common_rawio_test.py", line 122, in test_read_all
    reader.parse_header()
  File "/home/circleci/repo/neo/rawio/baserawio.py", line 151, in parse_header
    self._parse_header()
  File "/home/circleci/repo/neo/rawio/nixrawio.py", line 59, in _parse_header
    ch_name = da.metadata['neo_name']
  File "/home/circleci/repo/venv/lib/python3.6/site-packages/nixio/section.py", line 458, in __getitem__
    values = list(prop.values)
  File "/home/circleci/repo/venv/lib/python3.6/site-packages/nixio/property.py", line 243, in values
    v = self._read_old_values()
  File "/home/circleci/repo/venv/lib/python3.6/site-packages/nixio/property.py", line 234, in _read_old_values
    v = v["value"]
```

@achilleas-k Do you have an idea what's going wrong here?

**Comment by achilleas-k** [comment]
See https://github.com/NeuralEnsemble/python-neo/pull/767#issuecomment-554322399

I think that's the issue.

**Comment by JuliaSprenger** [comment]
fixed by completing #767
